cmd/cli/multiregion/multiregion.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright © 2023 SIL International
+*/
+
+package multiregion
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func SetupMultiregionCmd(parentCommand *cobra.Command) {
+	multiregionCmd := &cobra.Command{
+		Use:   "multiregion",
+		Short: "Tools for multiregion operation",
+		Long:  `Tools for multiregion setup, operation, fail-over, and fail-back`,
+	}
+
+	parentCommand.AddCommand(multiregionCmd)
+	InitStatusCmd(multiregionCmd)
+	InitSetupCmd(multiregionCmd)
+
+	var env string
+	multiregionCmd.PersistentFlags().StringVar(&env, "env", "prod", "Execution environment (default: prod)")
+	if err := viper.BindPFlag("env", multiregionCmd.PersistentFlags().Lookup("env")); err != nil {
+		log.Fatalln("Error: unable to bind flag:", err)
+	}
+
+	var region2 string
+	multiregionCmd.PersistentFlags().StringVar(&region2, "region2", "", "Secondary AWS region")
+	if err := viper.BindPFlag("region2", multiregionCmd.PersistentFlags().Lookup("region2")); err != nil {
+		log.Fatalln("Error: unable to bind flag:", err)
+	}
+}
+
+type PersistentFlags struct {
+	env             string
+	idp             string
+	org             string
+	readOnlyMode    bool
+	secondaryRegion string
+	token           string
+}
+
+func getPersistentFlags() PersistentFlags {
+	env := viper.GetString("env")
+	if env == "" {
+		log.Fatalln("no env is set, use --env to set the execution environment")
+	}
+
+	idp := viper.GetString("idp")
+	if idp == "" {
+		log.Fatalln("no IdP key is set, use --idp to set the IdP key")
+	}
+
+	org := viper.GetString("org")
+	if org == "" {
+		log.Fatalln("no Org is set, use --org to set the Terraform Cloud org")
+	}
+
+	token := viper.GetString("token")
+	if token == "" {
+		log.Fatalln("no Token is set, use --token to set the Terraform Cloud API token")
+	}
+
+	secondaryRegion := viper.GetString("region2")
+	if secondaryRegion == "" {
+		log.Fatalln("no secondary region is set, use --region2 to set the Terraform Cloud API token")
+	}
+
+	readOnlyMode := viper.GetBool("read-only-mode")
```

Comment:
I would be happy dropping "mode" from all of these, but you can leave it in if you prefer.